cicecore/cicedynB/general/ice_init.F90

```diff
@@ -2210,6 +2236,7 @@ subroutine input_data
 
  1000    format (a20,1x,f13.6,1x,a) ! float
  1002    format (a20,5x,f9.2,1x,a)
+ 1003    format (a20,1x,f16.14,1x,a)
```

Comment:
It seems like here, you might want to have a G or E formatted output for deltamin since it's very small (e-9, e-11).  Or multiple deltamin by 1e6 or 1e9 and then write out with F formatting?